src/schema/objects/metadata.yaml

```diff
@@ -2293,7 +2293,7 @@ SamplingFrequency:
 ScaleFactor:
   name: ScaleFactor
   description: |
-    Scale factor for each frame.
+    Scale factor for each frame. Values must be set if the imaging data (.nii) are unscaled, otherwise set to 1.
```

Comment:
Could you explain this a bit for me? I don't do PET, so I'm a little confused. It sounds like you're saying that this field is REQUIRED if the data are unscaled, but also required if they aren't.

Reply:
I'm going to have to refresh my memory on this as well, but I seem to recall that we ran into some double scaling occurring as a result of converting to nifti (this smells of ECAT). And we wanted to preserve that scaling history in some way in the spec.

Reply:
I can see how you would need the scaling factor in that case, but I'm still confused about what the update to the description is meant to convey.

Reply:
it makes explicit something that was implicit - some conversion wrote the scaling values in the json and applied to the data then what should a user do then? with that change we know

Reply:
I'm still confused. Perhaps the source of my confusion will be clearer if I break the added sentence down.

Part 1: "Values must be set if the imaging data (.nii) are unscaled"

My interpretation: If the data are unscaled, `ScaleFactor` is required.

Part 2: "otherwise set to 1."

My interpretation: If the data are scaled, `ScaleFactor` is required.

The combined interpretation would thus be, regardless of whether the data are scaled or not, `ScaleFactor` is required. But then, in the table, `ScaleFactor` isn't required, it's recommended.

Reply:
well yes - that's simpler to do so

Reply:
I think the concern is that by updating `ScaleFactor` to required will break the few currently passing datasets that are out there. @mnoergaard had some concerns regarding this so I will let him weigh in on this.

Reply:
You're right, that would break backwards compatibility. How about something like this?

> This field MUST be defined if the imaging data (.nii[.gz]) is scaled. If this field is not defined, then it is assumed that the scaling factor is 1. Defining this field when the scaling factor is 1 is recommended, for the sake of clarity.

Reply:
I'd be in favor of such a suggestion - thanks @tsalo! @CPernet what do you think of this?

Reply:
yep perfect thx @tsalo

Reply:
Updated [here](https://github.com/bids-standard/bids-specification/blob/6b946361322a3583d57fbc37b0d9d3f728c6d8fa/src/schema/objects/metadata.yaml#L2296), should be gtg.